miasm2/expression/expression.py

```diff
@@ -344,15 +343,21 @@ class ExprInt(Expr):
      - Constant 0x12345678 on 32bits
      """
 
-    def __init__(self, arg):
-        """Create an ExprInt from a numpy int
-        @arg: numpy int"""
-
-        if not is_modint(arg):
-            raise ValueError('arg must by numpy int! %s' % arg)
-
-        self._arg = arg
-        self._size = self.arg.size
+    def __init__(self, arg, size=None):
```

Comment:
As the function is refactored, maybe `arg` could be renamed to `number` or something similar